docs/hub/models-widgets-examples.md

```diff
@@ -0,0 +1,303 @@
+# Widget Examples
```

Comment:
Could we maybe do separate page for each widget example, and in the main page have an index similar to the one in http://hf.co/docs/hub, one per modality? I think that would be cooler and goes along our idea of split pages